Fabric.Authorization.AccessControl/src/app/modules/access-control/custom-group/custom-group.component.ts

```diff
@@ -58,15 +60,20 @@ export class CustomGroupComponent implements OnInit, OnDestroy {
     @Inject('IAccessControlConfigService') private configService: IAccessControlConfigService,
     private roleService: FabricAuthRoleService,
     private groupService: FabricAuthGroupService,
-    private edwAdminService: FabricAuthEdwadminService,
-    private idpSearchService: FabricExternalIdpSearchService
+    private edwAdminService: FabricAuthEdwAdminService,
+    private idpSearchService: FabricExternalIdpSearchService,
+    private currentUserService: CurrentUserService
   ) { }
 
   ngOnInit() {
+
     this.groupName = this.route.snapshot.paramMap.get('subjectid');
     this.grain = this.route.snapshot.paramMap.get('grain');
     this.securableItem = this.route.snapshot.paramMap.get('securableItem');
     this.editMode = !!this.groupName;
+    this.currentUserService.getPermissions().subscribe(p => {
+      this.missingManageAuthorizationPermission = !p.includes(`${this.grain}/${this.securableItem}.manageauthorization`);
```

Comment:
Should this check also check the roles associated to the custom group, if any? Since a custom group can be associated to roles across securables, we will deny the permission on the server if the user doesn't have manageauthorization permission for the securables tied to all the roles...